topdown/errors.go

```diff
@@ -65,6 +65,16 @@ func (e *Error) Error() string {
 	return msg
 }
 
+// GetCode returns code from Error structure
```

Comment:
Nit: Let's get rid of these getters. They're not required. You can refer to e.Code or e.Message.